docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3"
```

Comment:
i used this VSCode extension at my last co-op (https://code.visualstudio.com/docs/remote/containers) for coding inside our Docker container - not sure if it'd be useful here but worth checking out!

Reply:
Ham can you take me thru that sometime?